src/views/view.c

```diff
@@ -934,6 +934,8 @@ int dt_view_get_image_to_act_on()
 dt_view_surface_value_t dt_view_image_get_surface(int imgid, int width, int height, cairo_surface_t **surface,
                                                   const gboolean quality)
 {
+  const double tt = dt_get_wtime();
```

Comment:
We may want to get this only if needed so if DT_DEBUG_LIGHTTABLE is activated:

`if(darktable.unmuted & DT_DEBUG_LIGHTTABLE)`

Also since this is a timing we may want to activate it also if DT_DEBUG_PERF is on.

`if(darktable.unmuted & (DT_DEBUG_LIGHTTABLE | DT_DEBUG_PERF))`